pxr/usdImaging/usdviewq/stageView.py

```diff
@@ -1488,12 +1501,18 @@ def updateGL(self):
         playback.  The client driving playback at a particular rate should
         instead call updateForPlayback() to image the next frame."""
         if not self._dataModel.playing:
-            super(StageView, self).updateGL()
+            if PySideModule == 'PySide6':
+                super(StageView, self).update()
```

Comment:
Just inject `updateGL()` into QGLWidget ?  Not entirely satisfying as it's being deprecated, but I'm not sure atm whether there's a good reason we're calling `updateGL()` here rather than straight-up `update()`

Reply:
Thanks, done.